ui/apps/vibes/src/components/app-routes/index.tsx

```diff
@@ -84,13 +84,9 @@ const reportItemRoute = new Route({
 });
 
 const routeTree = rootRoute.addChildren([
-  overviewRoute,
-  moderationValueRoute,
-  moderatorsRoute,
-  viewModeratorRoute,
-  historyRoute,
-  historyItemRoute,
-  reportItemRoute,
+  overviewRoute.addChildren([moderationValueRoute]),
```

Comment:
where you have routes as children of other routes, remember to also provide the parent route, ex: for moderationValueRoute,  getParentRoute: () => overviewRoute,  and also adjust the path